src/main/java/org/opensearch/searchrelevance/rest/RestDeleteScheduledExperimentAction.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.searchrelevance.rest;
+
+import static java.util.Collections.singletonList;
+import static org.opensearch.rest.RestRequest.Method.DELETE;
+import static org.opensearch.searchrelevance.common.PluginConstants.DOCUMENT_ID;
+import static org.opensearch.searchrelevance.common.PluginConstants.SCHEDULED_EXPERIMENT_URL;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.action.DocWriteResponse;
+import org.opensearch.action.delete.DeleteResponse;
+import org.opensearch.core.action.ActionListener;
+import org.opensearch.core.rest.RestStatus;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.BytesRestResponse;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.searchrelevance.exception.SearchRelevanceException;
+import org.opensearch.searchrelevance.settings.SearchRelevanceSettingsAccessor;
+import org.opensearch.searchrelevance.transport.OpenSearchDocRequest;
+import org.opensearch.searchrelevance.transport.scheduledJob.DeleteScheduledExperimentAction;
+import org.opensearch.transport.client.node.NodeClient;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+/**
+ * Rest Action to facilitate requests to delete a scheduled experiment.
+ */
+@AllArgsConstructor
+public class RestDeleteScheduledExperimentAction extends BaseRestHandler {
+    private static final Logger LOGGER = LogManager.getLogger(RestDeleteScheduledExperimentAction.class);
+    private static final String DELETE_SCHEDULED_EXPERIMENT_ACTION = "delete_scheduled_experiment_action";
+    private SearchRelevanceSettingsAccessor settingsAccessor;
+
+    @Override
+    public String getName() {
+        return DELETE_SCHEDULED_EXPERIMENT_ACTION;
+    }
+
+    @Override
+    public List<Route> routes() {
+        return singletonList(new Route(DELETE, String.format(Locale.ROOT, "%s/{%s}", SCHEDULED_EXPERIMENT_URL, DOCUMENT_ID)));
```

Comment:
DELETE /experiment/{experiment_id}/schedule.  No need for job id since you can ONLY have one job per experiment.  You can't run an experiment nightly and on the hour.   That is part of our security/validation layer!  Even though under the covers it could be done...